src/annotator/sidebar.coffee

```diff
@@ -46,6 +46,8 @@ module.exports = class Sidebar extends Host
       @onProfileRequest = serviceConfig.onProfileRequest
       @onHelpRequest = serviceConfig.onHelpRequest
 
+    @onLayoutChange = config.onLayoutChange;
```

Comment:
My [CoffeeScript syntax highlighter](https://github.com/kchmck/vim-coffee-script) complains if lines end with semicolons. The compiler doesn't complain but we should omit them for consistency.